Exercise_4.py

```diff
@@ -1,17 +1,46 @@
-# Python program for implementation of MergeSort 
+# Python program for implementation of MergeSort
 def mergeSort(arr):
+    if len(arr)>1:
```

Comment:
Good Job! Please add time and space complexity and a few comments about your approach